src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,24 @@
+package calculator;
+
+public class StringCalculator {
+    private static final String ZERO = "0";
+    private final PositiveIntegers values;
+
+    public StringCalculator(PositiveIntegers values) {
+        this.values = values;
+    }
+
+    public static StringCalculator of(String str) {
+        if (str == null || str.isEmpty()) {
+            return new StringCalculator(new PositiveIntegers(PositiveInteger.of(ZERO)));
```

Comment:
0의 값을 가지는 PositiveInteger 객체를 미리 만들어두는것은 어떻게 생각하시나요? 😃 

```suggestion
            return new StringCalculator(new PositiveIntegers(PositiveInteger.ZERO));
```

Reply:
꼼꼼하게 봐주셔서 감사합니다~ 더 멋진 코드가 되었습니다